### ISSUE TITLE
Add alternative highlighting

### ISSUE THREAD

**Issue by Critical-Impact** [Critical-Impact/InventoryTools#8]
At present we just hue shift the colours of the inventory grid to show where a relevant item is, need to investigate other ways we could highlight a slot with a colour or perhaps even an icon.

**Comment by jjspace** [comment]
How about an outline or border to items? Playing with the tool I found the hue shift often washed out the image or was too dim to notice but a strong border might work?

Alternatively an option to dim everything that _doesn't_ show up in the search similar to how the Armory chest dims armor you can't equip to the current job

**Comment by Critical-Impact** [comment]
This isn't out yet but I've implemented a Global/Filter configuration option to invert the way highlighting works. So it'll colour everything that aren't the items found.
I've also added in new Global/Filter configuration option to only highlight on foreground filters when you actually start typing into one of the search fields.
I might get a release out soon then start to work on other ways to highlight. Getting the inverting certainly won't go amiss as any new ways I add of highlighting will still work with the inverting.

Appreciate the comment as well, the inverting makes sense
